mmv1/products/vpcaccess/Connector.yaml

```diff
@@ -136,6 +136,13 @@ properties:
       The fully qualified name of this VPC connector
     output: true
     ignore_read: true
+  - !ruby/object:Api::Type::Array
+    name: connectedProjects
+    description: |
+      List of projects using the connector.
+    output: true
+    unordered_list: true
```

Comment:
```suggestion

```

This property appears to be for fields that aren't outputs from the API - I suggest removing this line as it's not affecting how the provider code is built